### DISCUSSION TITLE
Netlify issues

### DISCUSSION THREAD

**Discussion by Swepool** [ivanhofer/typesafe-i18n#335] [Q&A] [answered]
I get this when deploying to netlify using their adapter, tried with the example page as well, seems to give me the same error. 

Is there anything I need to change to run it on netlify?

```
Require stack:
- /var/task/node_modules/typesafe-i18n/svelte/index.cjs
- /var/task/.netlify/server/chunks/i18n-svelte-e2aefd17.js
- /var/task/.netlify/server/entries/pages/__layout.svelte.js
- /var/task/.netlify/server/nodes/0.js
- /var/task/.netlify/functions-internal/render.js
- /var/task/render.js
- /var/runtime/UserFunction.js
- /var/runtime/index.js
```

**Comment by ivanhofer** [comment, marked as answer]
Hi @Swepool,

Which example have you tried to deploy?

**Reply by Swepool** [reply]
The sveltekit one ✌

**Reply by ivanhofer** [reply]
There are two SvelteKit example repositories 😅
Have you tried the TypeSchript or JavaScript repo?

**Reply by Swepool** [reply]
Ah that's right, sorry 😅  I tried the JS one with Netlify after my repo got the same error

**Reply by ivanhofer** [reply]
Could you please try to update SvelteKit to the newest version in your package.json? SvelteKit is still not stable and there could be a mismatch between the installed version and the newest netlify adapter.

**Reply by Swepool** [reply]
Ah, I'll give it a try!

**Reply by ivanhofer** [reply]
@Swepool I now finally got some time to take a look at the issue.
I made a fix in version `5.5.2` and deploying to netlify should now work too.

I have updated the example to the newest SvelteKit version and made those changes to use the netlify adapter:
https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit-jsdoc/compare/main...netlify
